.github/workflows/ci.yml

```diff
@@ -61,263 +62,263 @@ jobs:
   go_lint:
     uses: 'abcxyz/pkg/.github/workflows/go-lint.yml@main' # ratchet:exclude
     with:
-      go_version: '1.20'
+      go_version: '1.21'
```

Comment:
Should these use `${{env.go_version}}` rather than hardcoding 1.21 (throughout this file)?

Reply:
I believe capitalization matters, the env is UPPER and you gave lower, maybe try that?